src/main/java/CountryReader.java

```diff
@@ -0,0 +1,22 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+
+public class CountryReader {
+    public HashMap<String, Country> readCountries(String fileName) throws IOException {
```

Comment:
Jako typu zwracanego przez metodę używaj interfejsów zamiast implementacji, w tym przypadku Map. Pozwoli to na ukrycie implementacji i w razie potrzeby podmianę HashMap na np. `LinkedHashMap` bez ruszania reszty kodu